### ISSUE TITLE
path with spaces error

### ISSUE THREAD

**Issue by joneschunghk** [bmaltais/kohya_ss#2642]
```
16:15:05-647695 ERROR    The path in which this python code is executed contain one or many spaces. This is not supported for running kohya_ss GUI.
16:15:05-649695 ERROR    Please move the repo to a path without spaces, delete the venv folder and run setup.sh again.
16:15:05-650695 ERROR    The current working directory is: D:\AI\Stable Diffusion\kohya_ss
```
I updated from v22.6.0. Previous versions did not have this error. I don't want to rename my paths because all my stable diffusion related tools are there

**Comment by bmaltais** [comment]
Correct. This is new to this version. Spaces in path have been causing a lot of issues and I decided to just not support them. I welcome you to submit a PR to add support back to them once full testing on windows and Linux prove it will work flawlessly.

**Comment by OldGameRise** [comment]
In "kohya_ss\setup\validate_requirements.py" remove 'def check_path_with_space' function.
It will not check for spaces in path but I did not yet test this whether it will crash anything or not.

**Comment by bmaltais** [comment]
It will fail to run properly. If someone want to fix all the code to make it work, I will gladly take a PR to remove this restriction. Need to work for both windows and linux
